__tests__/integration/utils/test-utils-integration.js

```diff
@@ -794,18 +794,42 @@ export class TestUtils {
   static async waitNewBlock(currentHeight = null) {
     let baseHeight = currentHeight;
 
+    // This timeout is a protection, so the integration tests
+    // don't keep running in case of a problem
+    // After using the timeout as 120s, we had some timeouts
+    // because the CI runs in a free github runner
+    // so we decided to increase this timeout to 600s, so
+    // we don't have this error anymore
+    const timeout = 600000;
```

Comment:
Should this be a env variable? Maybe we can get it from the project settings in github

The idea is that if the github runner is slow for some issues on their side, we can just tune it accordingly without the need of a code change

What do you think?

Reply:
While I like this idea, a code change would always be necessary to modify the `env` variables for an action.

The main difference would be the modified file for adjustments: if we fetch this data from the env variables, we would need to change the `.github/workflows/integration-test.yml` instead of the `__tests__/integration/utils/test-utils-integration.js`

Reply:
Tulio is correct, we would just need to change a different file, the env var wouldn't help for your idea.

As discussed on Slack, if that was the case, we could close this comment.

Reply:
As we discussed in slack, we can just fetch it from the github variables and use it in the javascript file